src/main/getDiForUnitTesting.ts

```diff
@@ -55,6 +72,26 @@ export const getDiForUnitTesting = (
 
   di.preventSideEffects();
 
+  if (overrideHotbarStore) {
```

Comment:
Note: These ifs break Open Closed Principle, but so do the many overrides in this file. A better solution is in the horizon.